agent/ecscni/plugin.go

```diff
@@ -349,6 +389,32 @@ func (client *cniClient) createAppMeshConfig(cfg *Config) (string, *libcni.Netwo
 	return defaultAppMeshIfName, networkConfig, nil
 }
 
+func (client *cniClient) createBranchENINetworkConfig(cfg *Config) (string, *libcni.NetworkConfig, error) {
+
+	// cfg.ENIIPV4Address does not have a prefix length while BranchIPAddress expects a prefix length
```

Comment:
I believe I had to write this prefix length computation code when implementing the proof of concept. But I was hoping that we could have the control plane send the IP addresses in the correct format in production version. I'm sure there was a good reason why CP sends these addresses in the format it does today, but technically the prefix length is an attribute of the task IP address. On the other hand, we don't care what the prefix length of the gateway is; it is just the next hop of the default route.

Is this a good opportunity to iron out these issues?

Reply:
if we are going to change the format, backend will need to have some logic like "if this is not eni trunking task, use old format, else use new format". not sure if they will be happy with such logic. we can discuss with them about this though